hack/generator/pkg/astmodel/interface_implementation.go

```diff
@@ -57,6 +57,11 @@ func (iface *InterfaceImplementation) References() TypeNameSet {
 	return results
 }
 
+// FunctionCount returns the number of included functions
+func (iface *InterfaceImplementation) FunctionCount() int {
```

Comment:
Would it make more sense to just expose this as `Functions()` like exists on `ResourceType`? (Can we do that since this is also embedded there?)